### PR TITLE
SEO-2447 JS DropDownList

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @ChristoIssac ,
Please find changes made on JS DropDownList.
Regards,
@Scholar-jaruha

### CODE REVIEW

File: js/DropDownList/LoadOnDemand.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Load on deamand in DropDownList widget for Syncfusion Essential JS
-description: Learn here about Load on deamand in Syncfusion Essential JavaScript DropDownList widget, its elements, and more.
+title: Load on deamand in JS DropDownList | Syncfusion 
+description: Learn here about getting started with Syncfusion JS DropDownList control, its elements, and more functions.
```
Comment (Aishwarya-Ganesan) [inline js/DropDownList/LoadOnDemand.md:4]: functions word need to remove

File: js/DropDownList/LoadOnDemand.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Load on deamand in DropDownList widget for Syncfusion Essential JS
-description: Learn here about Load on deamand in Syncfusion Essential JavaScript DropDownList widget, its elements, and more.
+title: Load on deamand in JS DropDownList | Syncfusion
```
Comment (Aishwarya-Ganesan) [inline js/DropDownList/LoadOnDemand.md:3]: Load on deamand in JS DropDownList Control | Syncfusion  - Change like this

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: js/DropDownList/LoadOnDemand.md (hunk 1 above)
Comment (Scholar-jaruha) [reply js/DropDownList/LoadOnDemand.md]: resolved

File: js/DropDownList/LoadOnDemand.md (hunk 2 above)
Comment (Scholar-jaruha) [reply js/DropDownList/LoadOnDemand.md]: resolved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**